g15helper/g15helper.pro

```diff
@@ -5,58 +5,80 @@
 
 # Build rules for G15 helper daemon
 
-CONFIG += force-x86-toolchain
+win32:!CONFIG(g15-emulator) {
+  CONFIG += force-x86-toolchain
+}
 include (../compiler.pri)
 
 TEMPLATE = app
-CONFIG -= qt
 CONFIG *= debug_and_release warn_on
 TARGET = mumble-g15-helper
 
 HEADERS = g15helper.h
 
-win32 {
-  SOURCES = g15helper.c
-  RC_FILE = g15helper.rc
-  LIBS *= -llgLcd -ladvapi32 -lshell32 -luser32
-  QMAKE_LIBDIR *= "$$G15SDK_PATH/Lib/x86"
-  INCLUDEPATH *= "$$G15SDK_PATH/Src"
-  DEFINES *= WIN32
-  QMAKE_POST_LINK = $$QMAKE_POST_LINK$$escape_expand(\\n\\t)$$quote(mt.exe -nologo -updateresource:$(DESTDIR_TARGET);1 -manifest ../src/mumble/mumble.appcompat.manifest)
+CONFIG(g15-emulator) {
```

Comment:
So this is an either/or configuration? Either I get a daemon exe with real g15 OR the emu impersonating it? Probably ought to be in a comment.